doc/whatsnew/fragments/3391.extension

```diff
@@ -0,0 +1,6 @@
+Added ``consider-using-augmented-assign`` check for ``CodeStyle`` extension
+which flags ``x = x + 1`` to simplify to ``x += 1``.
+This check is disabled by default. To use it, add the msgid to ``enable``.
```

Comment:
```suggestion
This check is disabled by default. To use it, add the msgid to ``enable`` and load the code style extension with ``load-plugins=pylint.extensions.code_style``.
```
Might be obvious for long time users, but I think specifying it each time in release note is a good thing for new users. (I've learned about how to load extension embarrassingly late myself).